samples/README_Windows.md

```diff
@@ -0,0 +1,98 @@
+
+![Sample components diagram](sampledirstructure.png)
+
+All the samples that come with the Open Enclave SDK installation are all structured into two subdirectories (one for enclave and one for host) accordingly.
+
+| Files/dir        |  contents                                   |
+|:-----------------|---------------------------------------------|
+| enclave        | Files needed for building the sample enclave|
+| host           | Files needed for building the host          |
+
+### Prepare samples
+
+Building samples involves writing files into the working directory. You can do this in the directory in which you have installed your samples.
+You can also copy the samples from the location they were installed to another directory.
+
+```cmd
+xcopy  C:\openenclave\share\openenclave\samples C:\mysample
+```
+
+### How to build and run samples
+
+1. [x64 Native Tools Command Prompt for VS(2017 or 2019)](
+https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)
+Normally this is accessible under the `Visual Studio 2019 or 2017` folder in the Start Menu.
+
+2. Set OpenEnclave_DIR to the cmake directory in the Open Enclave SDK installation.
+
+As an example, if the Open Enclave SDK is installed to `C:\openenclave`, then you would set OpenEnclaveDIR as shown below
+
+```cmd
+set OpenEnclaveDIR=C:\openenclave\lib\openenclave\cmake
+```
+
+3. To build a sample using CMake, change directory to your target sample directory and execute the following commands:
+
+```cmd
+mkdir build
+cd build
+cmake .. -G Ninja -DNUGET_PACKAGE_PATH=C:\your\path\to\intel_and_dcap_nuget_packages
+ninja
+```
+
+4. To run the sample, use below:
+
+```cmd
+ninja run
+```
+
+## Samples
+
+The following samples demonstrate how to develop enclave applications using OE APIs. It's recommended to go through the following samples in the order listed.
+
+#### [HelloWorld](helloworld/README.md)
+
+- Minimum code needed for an OE app
+- Help understand the basic components an OE application
+- Demonstrate how to build, sign, and run an OE image
+
+#### [File-Encryptor](file-encryptor/README.md)
+
+- Shows how to encrypt and decrypt data inside an enclave
+- Uses AES mbedTLS API to perform encryption and decryption
+
+#### [Data-Sealing](data-sealing/README.md)
+
+- Introduce OE sealing and unsealing features 
+- Demonstrate how to use OE sealing APIs
+- Explore two supported seal polices
+  - OE_SEAL_POLICY_UNIQUE
+  - OE_SEAL_POLICY_PRODUCT
+
+#### [Remote Attestation](remote_attestation/README.md)
+
+- Explain how OE attestation works
+- Demonstrate an implementation of remote attestation between two enclaves running on different machines
+
+#### [Local Attestation](local_attestation/README.md)
```

Comment:
Is the attested_tls sample actually removed from the Windows version of the SDK package? I don't see where that's being explicitly dropped for the nuget package.

Reply:
It should be removed. Thanks for catching this. Will look.